lib/driver.js

```diff
@@ -59,14 +69,17 @@ export class ChromiumDriver extends BaseDriver {
     jsonwpRequiredCaps,
     w3cCapabilities,
   ) {
-    const [sessionId, caps] = /** @type {[string, ChromiumDriverCaps]} */(await super.createSession(jsonwpDesiredCapabilities, jsonwpRequiredCaps, w3cCapabilities));
-    await this.startChromedriverSession();
-    return [sessionId, caps];
+    const [sessionId,] = /** @type {[string, ChromiumDriverCaps]} */(await super.createSession(jsonwpDesiredCapabilities, jsonwpRequiredCaps, w3cCapabilities));
+    const returnedCaps = await this.startChromedriverSession();
+    if (returnedCaps.webSocketUrl) {
+      this._bidiProxyUrl = (/** @type {string|null} */(/** @type {unknown} */(returnedCaps.webSocketUrl)));
+    }
+    return [sessionId, returnedCaps];
   }
 
   getSessionCaps() {
     return Object.keys(this.opts).reduce((acc, capName) => {
-      if (STANDARD_CAPS_LOWER.has(capName) || capName.startsWith(CHROME_VENDOR_PREFIX) || capName.startsWith(EDGE_VENDOR_PREFIX)) {
+      if (STANDARD_CAPS_LOWER.has(capName.toLowerCase()) || capName.startsWith(CHROME_VENDOR_PREFIX) || capName.startsWith(EDGE_VENDOR_PREFIX)) {
```

Comment:
cap names are case sensitive, aren't they?

Reply:
mm, I needed this to fix an issue where caps were not properly forwarded to chromedriver.